env.js

```diff
@@ -1,5 +1,11 @@
+const dotenv = require('dotenv')
+
 if (!process.env.NODE_ENV) {
   throw new Error('Must set NODE_ENV')
 }
 
-require('dotenv').config({ path: `.env.${process.env.NODE_ENV}` })
+if (process.env.NODE_ENV !== 'prod') {
+  dotenv.config({ path: `.default.env` })
```

Comment:
I don't think we should use `.default.env` in our code. People will change this file and commit real credentials. Once the credentials are on GitHub it's really hard to erase them. It should be an example file. Maybe rename it to `.env.example`.